### PR TITLE
`exp`: better explain experiment names in `--rev`

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Follow-up to https://github.com/iterative/dvc.org/pull/3594#pullrequestreview-992934570

### CODE REVIEW

File: content/docs/api-reference/open.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ optimizes memory usage.
   (the current working directory tree is walked up to find it).
 
 - `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
-  a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
-  option is ignored. _Default_: `HEAD`.
+  a branch or tag name, commit hash, or [experiment name]). If `repo` is not a
```
Comment (pared) [inline content/docs/api-reference/open.md:62]: No link for `[experiment name]`

File: content/docs/api-reference/read.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ These are loaded to memory directly (without using any disc space).
   (the current working directory tree is walked up to find it).
 
 - `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
-  a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
-  option is ignored. _Default_: `HEAD`.
+  a branch or tag name, commit hash, or [experiment name]). If `repo` is not a
```
Comment (pared) [inline content/docs/api-reference/read.md:53]: No link for `experiment name`

**Review by pared — CHANGES_REQUESTED** [review]

File: content/docs/api-reference/open.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/api-reference/open.md:62]: Fixed

File: content/docs/api-reference/read.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/api-reference/read.md:53]: Fixed

**Comment by jorgeorpinel** [comment]
Sorry for delay @pared please approve + feel free to merge 🙂 or I will if you can't. Thanks

**Comment by jorgeorpinel** [comment]
Ping @pared @dberenbaum 🙏🏼

**Review by pared — APPROVED** [review]

**Comment by pared** [comment]
@jorgeorpinel sorry!

**Comment by jorgeorpinel** [comment]
No worries, thanks!

**Pull request merged**